src/main/java/tictac7x/charges/ChargesImprovedConfig.java

```diff
@@ -1869,6 +1884,13 @@ public interface ChargesImprovedConfig extends Config {
             section = debug
         ) default int getCoalBagCharges() { return Charges.UNKNOWN; }
 
+        @ConfigItem(
+            keyName = colossal_pouch,
```

Comment:
since the item is item that stores item, use the + "_storage" in here, should be String as well